### ISSUE TITLE
Default style not loading

### ISSUE THREAD

**Issue by spaquet** [Godofbrowser/vuejs-dialog#46]
main.js
```
import 'vuetify/dist/vuetify.min.css'
import 'vuejs-dialog/dist/vuejs-dialog.min.css'
```

However the style is not applied to dialogs when they are showing up. Any idea why since everything else  is working as expected?

**Comment by GergelyLakatos** [comment]
same here..

**Comment by gautamc1106** [comment]
same here

**Comment by Godofbrowser** [comment]
Can you try loading the script via html tag pending when this is resolved?

**Comment by gautamc1106** [comment]
Does not work!

**Comment by Godofbrowser** [comment]
Okay, can you help me? want to try reproduce the issue ... how are you using the lib?

**Comment by gautamc1106** [comment]
All I was doing initially is
`import 'vuejs-dialog/vuejs-dialog.min.css'`

and the following error wuld pop up:
`Module not found: Error: Can't resolve 'vuejs-dialog/vuejs-dialog.min.css' `

**Comment by gautamc1106** [comment]
I see the file in the module, but for some reason it is not detected

**Comment by gautamc1106** [comment]
I finally got it to work by loading the style sheet but the import fails

**Comment by Godofbrowser** [comment]
Actually, this is what's supposed to be the right path `vuejs-dialog/dist/vuejs-dialog.min.css`

**Comment by Godofbrowser** [comment]
What do you mean by "the import fails"? is your bundler configured to load css files?

**Comment by gautamc1106** [comment]
Yes. I did realised and that's how I fixed it. But when I use the same statement as 
`import "vuejs-dialog/dist/vuejs-dialog.min.css"` it fails to import it
